lib/librato.js

```diff
@@ -630,13 +630,15 @@ exports.init = function librato_init(startup_time, config, events, logger)
       tags = config.librato.tags;
     }
     
-    // Set global measurement tags if they are defined.
-    if (config.librato.writeToLegacy && Object.keys(config.librato.writeToLegacy).length) {
+    // Write legacy metrics if enabled.
+    if (config.librato.writeToLegacy) {
```

Comment:
Sure, but the check seems wrong for a boolean value `Object.keys(true).length === 0`